parent/pom.xml

```diff
@@ -225,6 +225,10 @@
         </profile>
         <profile>
             <id>staging</id>
+            <!-- TODO REMOVE ME -->
```

Comment:
Remove Me!

Reply:
Used to test the build-tools bits while in staging. Must be removed before merging this PR.

Reply:
Removed!